praw/models/comment_forest.py

```diff
@@ -72,6 +72,12 @@ def __len__(self) -> int:
         """Return the number of top-level comments in the forest."""
         return len(self._comments)
 
+    def __eq__(self, other: _CommentForest):
```

Comment:
I don't see a ton of value in comparing these lists. Let's hold off until someone has a compelling reason to need this functionality.

Reply:
What if I want to check if the comments in a submission have changed?

Reply:
Likely more efficient to compare `num_comments` in that case.

Reply:
What if we have the case where the number of comments didn't change (such as removal), but the comments did?

Reply:
You're right, `num_comments` wouldn't work for that. I just don't think that knowing if the comment sets are equal between two submission objects is something anyone really is going to do. And if they are, I'd like to know what they are actually trying to accomplish first.

Reply:
A lot of people want to monitor a submission in changes for comments. The previous comment shows why just monitoring the number of comments doesn't work, and it's mostly a convenience method.

Reply:
> A lot of people want to monitor a submission in changes for comments

I don't think making a new submission and comparing the comment forest is the best way to accomplish that. Building a custom stream for a single submission seems more viable.

Reply:
There exists no such stream. Is there a method to get the comments of a submission?

Reply:
https://www.reddit.com/dev/api/#GET_comments_{article} This API endpoint needs to be implemented. It seems that it returns a comment listing, but how do we deal with the MoreChildren?